### PR TITLE
Notice of archive

### PR DESCRIPTION
**Pull request by joshgowans**
Update intro paragraph to indicate that this document is no longer used to highlight contributors of future versions. Instead, users are encourage to look at the individual version release notes.

Overview
----------------------------------------
_A brief description of the pull request. Keep technical jargon to a minimum. Hyperlink relevant discussions._

Before
----------------------------------------
_What is the old user-interface or technical-contract (as appropriate)?_
_For optimal clarity, include a concrete example such as a screenshot, GIF ([LICEcap](http://www.cockos.com/licecap/), [SilentCast](https://github.com/colinkeenan/silentcast)), or code-snippet._

After
----------------------------------------
_What changed? What is new old user-interface or technical-contract?_
_For optimal clarity, include a concrete example such as a screenshot, GIF ([LICEcap](http://www.cockos.com/licecap/), [SilentCast](https://github.com/colinkeenan/silentcast)), or code-snippet._

Technical Details
----------------------------------------
_If the PR involves technical details/changes/considerations which would not be manifest to a casual developer skimming the above sections, please describe the details here._

Comments
----------------------------------------
_Anything else you would like the reviewer to note_

### CODE REVIEW

File: CONTRIBUTORS.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-The following people and organizations sponsored and/or contributed new and improved features to the project.
+As of CiviCRM 5.25, contributor recognition is viewable in the release notes of each individual version. This file represents an archive of recogntion for people and organizations that have sponsored and/or contributed to the project prior to version 5.25.
```
Comment (demeritcowboy) [inline CONTRIBUTORS.txt:1]: Small typo "recogntion" => "recognition"

**Comment by demeritcowboy** [comment]
Also this is based against 5.19? Should be 5.25?

**Comment by colemanw** [comment]
Tried changing the base but it created a bunch of merge conflicts. I suggest you close this and re-do it against master @joshgowans

**Comment by eileenmcnaughton** [comment]
@agh1

**Comment by joshgowans** [comment]
Sorry, wasn't paying attention. I'll close and redo on master.

**Pull request closed without merging**